[ci] - shellCheck test seems to need some tweeking

CI output:
```
Run cd ../csdiff && sudo make && sudo make install && cd -
make: *** No targets specified and no makefile found.  Stop.
Error: Process completed with exit code 2.
```

--- a/.github/workflows/shellcheck_test.yml
+++ b/.github/workflows/shellcheck_test.yml
@@ -16,7 +16,7 @@ jobs:
       - name: Install dependencies
         run: sudo apt update && sudo apt-get install -y cmake help2man libboost-dev libboost-filesystem-dev libboost-program-options-dev libboost-python-dev libboost-regex-dev tree
       - name: Clone csdiff repository
-        run: cd ../ && git clone --depth=1 https://github.com/kdudka/csdiff.git && cd -
+        run: cd ../ && git clone --depth=1 https://github.com/csutils/csdiff.git && cd -
       - name: Build and install csdiff
         run: cd ../csdiff && sudo make && sudo make install && cd -
       - name: Repository checkout

